src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
```

Comment:
Ensure that the `ul` element exists in the HTML and contains `li` elements with the necessary `data-*` attributes (e.g., `data-salary`, `data-position`, `data-age`) for the script to function correctly.